android/src/main/java/com/swmansion/rnscreens/ScreenWindowTraits.kt

```diff
@@ -185,6 +187,19 @@ object ScreenWindowTraits {
         window.navigationBarColor = color
     }
 
+    internal fun setNavigationBarTranslucent(screen: Screen, activity: Activity?) {
+        if (activity == null) {
+            return
+        }
+
+        val window = activity.window
+
+        val screenForNavBarTranslucent = findScreenForTrait(screen, WindowTraits.NAVIGATION_BAR_TRANSLUCENT)
+        val translucent = screenForNavBarTranslucent?.isNavigationBarTranslucent ?: false
+
+        WindowCompat.setDecorFitsSystemWindows(window, !translucent)
```

Comment:
I'd add a comment here of what this code does exactly.

Reply:
I meant something like It makes the content go under the navigationBar 😅 Does it make the content go under status bar also e.g. when there is no header displayed?

Reply:
https://github.com/software-mansion/react-native-screens/pull/2152/commits/950c8e34ba76f5a73fbf8a54690454de8b0b81ae
I updated the comment, let me know if it's clear now

Reply:
Looks good now. What about the second part: `Does it make the content go under status bar also e.g. when there is no header displayed?`?